components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -296,6 +306,12 @@ - (void)layoutSubviews {
   CGRect textFrame = UIEdgeInsetsInsetRect(self.bounds, textInsets);
   textFrame.origin.x += _leadingButtonBar.frame.size.width;
   textFrame.size.width -= _leadingButtonBar.frame.size.width + _trailingButtonBar.frame.size.width;
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
```

Comment:
If we are subtracting the leading/trailingButtonBar against textFrame above, which have already been adjusted by safeAreaInsets above, do we need to repeat the adjustment here?

Reply:
`textFrame.origin.x += _leadingButtonBar.frame.size.width` doesn't take into account the origin of the _leadingButtonBar frame :/ I'm afraid that CGRectGetMinX on that frame wouldn't be correct as the leadingButtonBar will be on the right for RTL languages.

Reply:
Ack.